### PR TITLE
Add css to button, label and textarea.

### PR DESCRIPTION
**Pull request by migacz1125**
Add css to button, label and textarea.

### CODE REVIEW

File: src/app.css [hunk 1]
```diff
@@ -50,9 +50,9 @@ hr {
     font-size: 1.3em;
     width: 100%;
     resize: none;
-    overflow: hidden;
-    margin: 0px;
-    padding: 0px;
+    -webkit-box-shadow: none;
+	-moz-box-shadow: none;
+	box-shadow: none;
```
Comment (piotrwitek) [inline src/app.css:53]: please remove vendor prefixes, should be added automatically by build tool like autoprefixer

File: src/app.css [hunk 2]
```diff
@@ -62,6 +62,10 @@ hr {
 .title__text {
     font-size: 2em;
     width: 100%;
+    margin-top: 10px;
+    -webkit-box-shadow: none;
```
Comment (piotrwitek) [inline src/app.css]: vendor prefixes again

File: src/app.css (hunk 1 above)
Comment (migacz1125) [inline src/app.css:53]: done

File: src/app.css (hunk 2 above)
Comment (migacz1125) [inline src/app.css]: done

**Pull request merged**